### PR TITLE
fix usage text + remove inetface file from build

### PR DESCRIPTION
**Pull request by cherokee94**

### CODE REVIEW

File: docs/modules/usage/nav.adoc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 * xref:overview.adoc[]
 ** xref:concepts.adoc[]
-** xref:user_interface_overview.adoc[]
```
Comment (arufanov) [inline docs/modules/usage/nav.adoc:3]: @cherokee94 
* should we remove file: axion/docs/modules/usage/pages/user_interface_overview. I suppose yes.

Comment (arufanov) [reply docs/modules/usage/nav.adoc:3]: @cherokee94 
I have already done

**Comment by cherokee94** [comment]
@arufanov removed the "allow", you can merge

**Review by arufanov — APPROVED** [review]
@cherokee94 
* thanks!

**Pull request merged**